client/my-sites/plugins/main.jsx

```diff
@@ -488,86 +488,59 @@ export class PluginsMain extends Component {
 						</div>
 					</FixedNavigationHeader>
 				) }
-				<div className={ classNames( { 'plugins__top-container': isJetpackCloud } ) }>
-					<div
-						className={ classNames( {
-							'plugins__content-wrapper': isJetpackCloud,
-						} ) }
-					>
-						{ isJetpackCloud && (
-							<div className="plugins__page-title-container">
-								<h2 className="plugins__page-title">{ pageTitle }</h2>
-								<div className="plugins__page-subtitle">
-									{ this.props.selectedSite
-										? this.props.translate( 'Manage all plugins installed on %(selectedSite)s', {
-												args: {
-													selectedSite: this.props.selectedSite.domain,
-												},
-										  } )
-										: this.props.translate( 'Manage plugins installed on all sites' ) }
-								</div>
+				<div
+					className={ classNames( 'plugins__top-container', {
+						'plugins__top-container-wp': ! isJetpackCloud,
+					} ) }
+				>
+					<div className="plugins__content-wrapper">
+						<div className="plugins__page-title-container">
+							<h2 className="plugins__page-title">{ pageTitle }</h2>
+							<div className="plugins__page-subtitle">
+								{ this.props.selectedSite
+									? this.props.translate( 'Manage all plugins installed on %(selectedSite)s', {
+											args: {
+												selectedSite: this.props.selectedSite.domain,
+											},
+									  } )
+									: this.props.translate( 'Manage plugins installed on all sites' ) }
 							</div>
-						) }
-						<div
-							className={ classNames( 'plugins__main', {
-								'plugins__jetpack-cloud': isJetpackCloud,
-							} ) }
-						>
+						</div>
+
+						<div className="plugins__main plugins__main-updated">
 							<div className="plugins__main-header">
 								<SectionNav
-									applyUpdatedStyles={ isJetpackCloud }
+									applyUpdatedStyles
```

Comment:
We are now using this new style in Agency Dashboard and Plugin Management. But, the plan is to use this throughout Jetpack Cloud.

Reply:
Got it! It is still a bit confusing because `<SectionNav />` is used quite a lot (I think) in WordPress.com.

Maybe we can document this in the README file of the SectionNav component in this case?

Reply:
Done - https://github.com/Automattic/wp-calypso/pull/67910/commits/059f630ec4da1ae8ed87a7f15bee6f75f490a401